client/landing/jetpack-cloud/components/backup-activity-log/backup-date-range-selector/placeholder.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { useTranslate } from 'i18n-calypso';
+import React, { FunctionComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@automattic/components';
+
+const BackupsDateRangeSelectorPlaceholder: FunctionComponent = () => {
+	const translate = useTranslate();
+
+	return (
+		<Button className="backup-date-range-selector__placeholder" disabled>
+			{ translate( 'Date range' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **126** times:
`translate( 'Date range:' )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-cloud-basic-activity-log)